src/app/window.cpp

```diff
@@ -40,7 +40,7 @@ Dash::Dash(Arbiter &arbiter)
     layout->addLayout(this->rail.layout);
     layout->addLayout(this->body.layout);
 
-    connect(&this->rail.group, QOverload<int>::of(&QButtonGroup::buttonPressed), [this](int id){
+    connect(&this->rail.group, &QButtonGroup::idPressed, [this](int id){
```

Comment:
this change breaks compatibility with older qt versions (and why the build fails in github) - I'm all for updating this once the pi's update the qt version tho!